manifests/jobs.pp

```diff
@@ -0,0 +1,22 @@
+# Configure dynflow
+class foreman::jobs {
```

Comment:
Bikeshedding: Should this be called `foreman::dynflow`?

Reply:
We would have to rename the attributes as well then :worried: